index.js

```diff
@@ -116,6 +126,13 @@ function runWebpack (config, opts) {
   })
 }
 
+function extractSize (stat, opts) {
+  let name = `${ stat.compilation.outputOptions.filename }`
+  name += opts.config ? '' : '.gz'
```

Comment:
Will it used gzipped size in custom config?

Reply:
It will use the output file name specified in the desired webpack configuration. So if you specify it, it will pick it up.

Reply:
What users need to do to show gzipped size?